modules/compiler/utils/include/compiler/utils/metadata.h

```diff
@@ -249,15 +249,15 @@ multi_llvm::Optional<std::array<uint64_t, 3>> parseRequiredWGSMetadata(
 ///
 /// @return The maximum work dimension or llvm::None if there is no such
```

Comment:
are all these comments now wrong - they don't return llvm::None an more but std::nullopt?

Reply:
oh, yes, good spot. I was only grepping for `multi_llvm::`

Reply:
done